target/linux/ramips/dts/mt7620a_netgear_ex6120.dts

```diff
@@ -0,0 +1,58 @@
+/* This file is released into the public domain */
+
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6120", "ralink,mt7620a-soc";
+	model = "Netgear EX6120";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
```

Comment:
Did You think about using amber power led for failsafe and upgrade?

Reply:
http://www.downloads.netgear.com/files/GDC/EX6120/EX6120_UM_25Aug2015.pdf
![image](https://user-images.githubusercontent.com/12028138/77834292-a964e080-717e-11ea-9800-6fa5a39bcbaa.png)
`led-boot` should be amber too.